// playground/grpc-related
syntax = "proto3";

package deepl;

service Deepl{
  // Deepl rpc service.
 rpc GetServerResponse(Message) returns (MessageResponse) {}
}

message Message{
 string message = 1;
 string from_lang = 2;
 string to_lang = 3;
}

message MessageResponse{
 string message = 1;
 bool received = 2;
}
